templates/forum/dashboard.haml

```diff
@@ -40,12 +41,36 @@
 							            Aucune compétence liée
 							        {% endif %}
 						        %td {{thread.modified_date}}
-		        			</tr>
+		        			
 			    	{% else %}
 			    		%tr
 			    			%td Aucune conversation trouvée
+			    			%td
```

Comment:
Use <td> with colspan=4 to expand the column to 4 columns instead of creating 4 columns

Reply:
https://developer.mozilla.org/fr/docs/Web/HTML/Element/td

Reply:
https://developer.mozilla.org/fr/docs/Web/HTML/Element/td